docs/concepts/dotnet-standard-library.md

```diff
@@ -0,0 +1,130 @@
+.NET Standard Library
+=====================
+
+The .NET Standard Library is a formal specification of .NET APIs that are intended to be available on all .NET runtimes. The motivation behind the Standard Library is establishing greater uniformity in the .NET ecosystem. [ECMA 335](https://github.com/dotnet/coreclr/blob/master/Documentation/project-docs/dotnet-standards.md) continues to establish uniformity for .NET runtime behavior, but there is no similar spec for the .NET Base Class Libraries (BCL) for .NET library implementations. 
```

Comment:
Nit: ECMA 335 did include the API set for the core assemblies. But it's hopelessly out of date.